### PR TITLE
refactor(output): render output via a utility wrapper

### PR DESCRIPTION
**Pull request by ta2edchimp**
Creates a utility wrapper for `cliui` and let's `bin/diff` and `bin/find` use it.

### CODE REVIEW

File: src/lib/cli-util.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+var size = require('window-size')
+var availableWidth = size.width || /*istanbul ignore next */ 80
```
Comment (sarbbottam) [inline src/lib/cli-util.js]: I like these sort of `/*istanbul ignore next */`, I personally don't write test for these default conditions 👍

File: src/lib/cli-util.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  var width = _output.reduce(
+    function getMaxWidth(previous, current) {
+      return Math.max(padding.left + current.length + padding.right, previous)
+    }, 0)
+
+  var _columns = columns || Math.floor(availableWidth / width)
+  var cellMapper = getOutputCellMapper(Math.floor(availableWidth / _columns), padding)
+
+  while (_output.length) {
+    ui.div.apply(ui, _output.splice(0, _columns).map(cellMapper))
```
Comment (sarbbottam) [inline src/lib/cli-util.js]: Could you help me understand, what push is supposed to do? Is `ui.div.apply` writing to the `STDOUT`? If so, then why there is a separate function `write`.

Comment (sarbbottam) [inline src/lib/cli-util.js]: I see what its is supposed to from the test, never mind. I was not aware that `ui.div.apply` is storing the information.

**Comment by sarbbottam** [comment]
LGTM.

File: src/lib/cli-util.js (hunk 2 above)
Comment (ta2edchimp) [inline src/lib/cli-util.js]: Yeah, that gave me a bit of a headache, I was hoping that I could somehow flush the output earlier, but after a quick glance over `cliui`'s sources, I guess this is not possible without creating a new instance of it every time the function now called `push` is invoked.

Maybe this is something to keep in mind for later, if we'd have real hard calculations going on, I'd prefer to print stuff asap, but in this case I guess it's of minor importance.

**Pull request merged**